hmy/downloader/beaconhelper.go

```diff
@@ -120,7 +120,9 @@ func (bh *beaconHelper) insertLastMileBlocks() (inserted int, bn uint64, err err
 			bn--
 			return
 		}
-		if err = bh.ih.verifyAndInsertBlock(b); err != nil {
+		// TODO: Instruct the beacon helper to verify signatures. This may require some forks
```

Comment:
no verification now?

Reply:
No. Since the block from pub-sub message does not contain `commitSigAndBitmaps` field, thus the signature verification of the this block cannot be fulfilled. Better add signature in pub-sub message in beacon client topic later. A network fork is expected.

Reply:
insertChain will verify the signature for previous block. But not this block. Definitely it's better to verify for this block before commit like we recently added for the sync logic.

Reply:
But we can do it as another task.